quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/ConfigCategory.java

```diff
@@ -8,10 +8,11 @@
     FEATURE("Feature", 40),
     HOSTNAME("Hostname", 50),
     HTTP("HTTP/TLS", 60),
-    METRICS("Metrics", 70),
-    PROXY("Proxy", 80),
-    VAULT("Vault", 90),
-    LOGGING("Logging", 100),
+    HEALTH("Health", 70),
```

Comment:
Perhaps one meta-category "Observability" instead of having multiple categories with only one option as it is now? @stianst @pedroigor wdyt?

Reply:
No strong opinions, but I like how it looks in the approval tests result now.

Reply:
+1 on strong opinions, absolutely not a blocker for me :)